builtins/amp-video.js

```diff
@@ -49,7 +73,7 @@ export function installVideo(win) {
         assertHttpsUrl(this.element.getAttribute('src'), this.element);
       }
       this.propagateAttributes(
-          ['src', 'controls', 'autoplay', 'muted', 'loop', 'poster'],
+          ['src', 'controls', 'autoplay', 'muted', 'loop', 'poster', 'preload'],
```

Comment:
No, the goal for the validator to allow it for the `amp-video` tag. I understand you want to allow it given this code:
```
this.element.getAttribute('preload')
```